recipes/repoze.who-friendlyform/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "repoze.who-friendlyform" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 77138a94b11e8f846889fd7e025d6914e5966125869f6d75ef216801e29cdf13
+  patches:
+    - patch3.patch
```

Comment:
Could you please add a comment here explaining the reason for this patch?

Reply:
There were imports that were for only for python 2.x (importing from urlparse & urllib) as well as use of a deprecated function from zope.interface

The patch imports from the python 3+ library (urllib.parse) if the python 2.x import fails, and uses an updated function from zope.interface that is backwards compatible